includes/Helpers/ProductTabsMetaHandler.php

```diff
@@ -0,0 +1,115 @@
+<?php
+/**
+ * Product Tabs Meta Handler
+ *
+ * @since 1.9.1-dev.1
+ */
+namespace WooCommerceCustomProductTabsLite\Helpers;
+
+use WC_Product;
+
+class ProductTabsMetaHandler
+{
+	const PRODUCT_TABS_META_KEY = '_wc_custom_product_tabs_lite_product_tabs';
+	const LEGACY_PRODUCT_TABS_META_KEY = 'frs_woo_product_tabs';
+
+	public function __construct()
+	{
+		$this->addHooks();
+	}
+
+	public function addHooks()
+	{
+		add_filter('get_post_metadata', [$this, 'maybeConvertLegacyProductTabsMeta'], 10, 5);
+	}
+
+	/**
+	 * Gets product meta.
+	 *
+	 * @param WC_Product $product
+	 * @return mixed
+	 */
+	public function getMeta(WC_Product $product, string $context = 'edit')
+	{
+		$this->maybeMigrateLegacyMeta($product);
+
+		$meta = $product->get_meta(static::PRODUCT_TABS_META_KEY, true, $context);
+
+		if (is_string($meta)) {
+			return json_decode($meta, true);
+		}
+
+		return $meta;
+	}
+
+	/**
+	 * Updates product tabs meta.
+	 *
+	 * @param WC_Product $product
+	 * @param array      $meta
+	 * @return void
+	 */
+	public function updateMeta(WC_Product $product, array|string $meta) : void
+	{
+		$product->update_meta_data(self::PRODUCT_TABS_META_KEY, json_encode(maybe_unserialize($meta)));
```

Comment:
```suggestion
		$product->update_meta_data(static::PRODUCT_TABS_META_KEY, json_encode(maybe_unserialize($meta)));
```

You used `static` elsewhere. Just keeping it consistent.

Reply:
Done in 2c9639daeefb84d2e097bbb151478039dda72894.